CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ Subsequently, the date entry follows **YYYY-MM-DD** format in accordance with th
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ---
+## [v0.1.2] (2020-11-26)
```

Comment:
is it correct?

Reply:
It is, though I'd prefer to add a link to v0.1.2 in the file (at the end, just like the other versions are linked to specific commits), and also add notes for v0.1.1, but it isn't critical I guess.

Reply:
@campfire5 ⬆️

Reply:
okay, there is no version or update for this version so there is no need of linking it, but you link it anyway if you have the desire